app/src/main/res/layout/fragment_mercadopago.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
xmlns in new line

Reply:
oh, my reformat code put that there, but ok 🤷